### ISSUE TITLE
Gradle - Cannot expand ZIP 'build\resources\aot' (Windows only)

### ISSUE THREAD

**Issue by Vity01** [spring-projects-experimental/spring-native#1211]
`gradlew.bat nativeBuild` fails with error:

```
Execution failed for task ':nativeBuildClasspathJar'.
> Cannot expand ZIP 'C:\develope\springnative\build\resources\aot' as it is not a file.
```
It's working fine on MacOs. 

- graalvm-ce-java11-21.3.0 with native-image extensions installed
- `id 'org.springframework.experimental.aot' version '0.10.5'`
- `id 'org.springframework.boot' version '2.5.6'`

Simple test app attached
[springnative.zip](https://github.com/spring-projects-experimental/spring-native/files/7467984/springnative.zip)

**Comment by sdeleuze** [comment]
Confirmed, I will raise that point with https://github.com/graalvm/native-build-tools team and maybe create a related issue there.

Seems to be triggered by our (IMO legit) configuration like below on Windows with native build tools 0.9.4+:
```
nativeBuild {
	classpath processAotResources.outputs, compileAotJava.outputs
}
nativeTest {
	classpath processAotTestResources.outputs, compileAotTestJava.outputs
}
```

I attach an updated sample with the right repository configuration.
[springnative2.zip](https://github.com/spring-projects-experimental/spring-native/files/7481910/springnative2.zip)

**Comment by sdeleuze** [comment]
Blocked by https://github.com/graalvm/native-build-tools/issues/152.

**Comment by sdeleuze** [comment]
Expected to be fixed fixed in Spring Native 0.11, I can't upgrade 0.10 to the latest Native Build Tools because Spring Boot 2.5 still uses JUnit 5.7, so please upgrade to fix this one.
